src/parser/transform/statement/transform_alter_table.cpp

```diff
@@ -23,19 +24,42 @@ unique_ptr<AlterStatement> Transformer::TransformAlter(duckdb_libpgquery::PGNode
 		case duckdb_libpgquery::PG_AT_AddColumn: {
 			auto cdef = (duckdb_libpgquery::PGColumnDef *)command->def;
 			auto centry = TransformColumnDefinition(cdef);
+
+			bool default_constraint_set = false;
+			unique_ptr<Constraint> generated_constraint = nullptr;
```

Comment:
`unique_ptr` is always instantiated to `nullptr`, so this assignment is not necessary